src/Payload.php

```diff
@@ -412,8 +408,10 @@ public function jsonSerialize()
     {
         $payload = self::getDefaultPayloadStructure();
 
-        if ($this->alert instanceof Alert || is_string($this->alert)) {
+        if ($this->alert instanceof Alert) {
             $payload[self::PAYLOAD_ROOT_KEY]->{self::PAYLOAD_ALERT_KEY} = $this->alert;
+        } elseif(is_string($this->alert)){
+            $payload[self::PAYLOAD_ROOT_KEY]->{self::PAYLOAD_ALERT_KEY} = json_decode($this->alert, true);
```

Comment:
What if `$this->alert` is a string, but it is not valid json? Seems we need one more check - if it is a valid json then use json_decode

Reply:
Yep, I will add check.